src/Adapter/Sftp.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Bolt\Filesystem\Adapter;
+
+use League\Flysystem\Config;
+use League\Flysystem\Sftp\SftpAdapter;
+
+/**
+ * SFTP adapter.
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class Sftp extends SftpAdapter
+{
+    /** @var int */
+    protected $filePerm = 0644;
+    /** @var int */
+    protected $directoryPerm = 0755;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct(array $config)
+    {
+        $this->configurable[] = 'filePerm';
```

Comment:
`configurable` just calls setters for config array. There is no `setFilePerm` so this does nothing, also not needed as mentioned above.